File: P68935.py
def solution(n):
    answer = 0
    
    result = ""
    
    while n:
        result += str(n%3)
        n //= 3
    
    tmp = 1
    for digit in result[::-1]:
        answer += int(digit) * tmp
        tmp *= 3
    
    
    return answer

File: P68935.java
class P68935 {
    public int solution(int n) {
        int answer = 0;
        String result = "";
        
        while (n > 0) {
            result += Integer.toString(n%3);
            n /= 3;
        }
        
        int tmp = 1;
        for (int i=result.length()-1; i>=0; i--) {
            char digit = result.charAt(i);
            answer += Character.getNumericValue(digit) * tmp;
            tmp *= 3;
        }
        return answer;
    }
}